dub.sdl

```diff
@@ -9,9 +9,9 @@ lflags "-L/usr/local/opt/llvm/lib" platform="osx"
 lflags "-L/usr/lib/llvm-3.9/lib" platform="posix"  # travis
 lflags "/LIBPATH:C:\\PROGRA~1\\LLVM\\lib" platform="windows"
 
-
 configuration "library" {
     targetType "library"
+    targetName "libclang-d"
```

Comment:
Otherwise if you build the lib and then the unittests, the unittests try to link to the lib instead of the real libclang